README.md

```diff
@@ -33,7 +33,7 @@ Enlabeler groups issue labels by color and description prefixes as shown in the
 Group | Subgroup | Label |
 --- | --- | ---
 Priority Level | | ![example label](https://labl.es/svg?text=priority:%20critical&bgcolor=9F0000) ![example label](https://labl.es/svg?text=priority:%20high&bgcolor=E30303) ![example label](https://labl.es/svg?text=priority:%20medium&bgcolor=ff6666) ![example label](https://labl.es/svg?text=priority:%20low&bgcolor=ffb3b3) 
-Status | | ![example label](https://labl.es/svg?text=blocked&bgcolor=1C6F87) ![example label](https://labl.es/svg?text=blocker&bgcolor=1C6F87) ![example label](https://labl.es/svg?text=stalled&bgcolor=8edee7) ![example label](https://labl.es/svg?text=question&bgcolor=8edee7) ![example label](https://labl.es/svg?text=needs%20revision&bgcolor=8edee7)  ![example label](https://labl.es/svg?text=needs%20estimate&bgcolor=8edee7) ![example label](https://labl.es/svg?text=has%20workaround&bgcolor=8edee7) ![example label](https://labl.es/svg?text=changes%20requested&bgcolor=8edee7) ![example label](https://labl.es/svg?text=duplicate&bgcolor=8edee7)
+Status | | ![example label](https://labl.es/svg?text=blocked&bgcolor=1C6F87) ![example label](https://labl.es/svg?text=blocker&bgcolor=1C6F87) ![example label](https://labl.es/svg?text=stalled&bgcolor=8edee7) ![example label](https://labl.es/svg?text=question&bgcolor=8edee7) ![example label](https://labl.es/svg?text=needs%20revision&bgcolor=8edee7)  ![example label](https://labl.es/svg?text=needs%20estimate&bgcolor=8edee7) ![example label](https://labl.es/svg?text=has%20workaround&bgcolor=8edee7) ![example label](https://labl.es/svg?text=changes%20requested&bgcolor=8edee7) ![example label](https://labl.es/svg?text=duplicate&bgcolor=8edee7) ![recommendation](https://labl.es/svg?text=recommendation&bgcolor=8edee7)
```

Comment:
We should probably add descriptive alt text to the other example labels as well.